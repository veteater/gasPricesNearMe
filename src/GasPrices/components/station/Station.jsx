import React from 'react';

const Station = () => {
  return (
    <React.Fragment>
      <span className="logo">Logo</span>
      <div className="info">
        <h2>title</h2>
        <ul>
          <li>Regular: $0.00</li>
          <li>Premium: $0.00</li>
          <li>Diesel: $0.00</li>
        </ul>
        <span>Directions</span>
      </div>
    </React.Fragment>
  );
};

export default Station;