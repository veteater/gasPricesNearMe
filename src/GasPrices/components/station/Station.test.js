import React from 'react';
import { shallow } from 'enzyme';
import Station from './Station';

describe('Station', () => {
  it('renders without crashing', () => {
    const component = shallow(<Station />);
    expect(component).toMatchSnapshot();
  });
});
