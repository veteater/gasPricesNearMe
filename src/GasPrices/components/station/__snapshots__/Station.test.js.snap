// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Station renders without crashing 1`] = `
<Fragment>
  <span
    className="logo"
  >
    Logo
  </span>
  <div
    className="info"
  >
    <h2>
      title
    </h2>
    <ul>
      <li>
        Regular: $0.00
      </li>
      <li>
        Premium: $0.00
      </li>
      <li>
        Diesel: $0.00
      </li>
    </ul>
    <span>
      Directions
    </span>
  </div>
</Fragment>
`;
